openwisp_controller/pki/migrations/0008_auto_20190106_1928.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.0.9 on 2019-01-06 18:28
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pki', '0007_default_groups_permissions'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='ca',
+            name='passphrase',
+        ),
+        migrations.RemoveField(
+            model_name='cert',
+            name='passphrase',
+        ),
+    ]
```

Comment:
this should not be here, you should use the dev version of django-x509 in your dev env, did you follow the pipenv instructions to install the dev environment?

Reply:
@nemesisdesign I have created new env following instructions and I installed `openwisp_users` by pip:  `pip install -e git+git://github.com/openwisp/openwisp-users#egg=openwisp_users` because version of it installed by instructions was too old and I received following information:
```
Your models have changes that are not yet reflected in a migration, and so won't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
```

so I runned `manage.py makemigrations` and the same migration file was created

Reply:
and my version of `django-x509` is `0.4.1`

Reply:
BTW sorry that I forgotten about creating new branch :anguished:

Reply:
please simply remove this file, I'm not sure how to help you to set up your dev env properly because I'm not using pipenv. I'm sure this migration file must not be here.

Reply:
> @nemesisdesign I have created new env following instructions and I installed `openwisp_users` by pip: `pip install -e git+git://github.com/openwisp/openwisp-users#egg=openwisp_users` because version of it installed by instructions was too old and I received following information:

@oltarzewskik So you do use `pipenv`, right? Then you also need to use `pipenv install` to install that special version of `openwisp-users`.

Reply:
@nemesisdesign I think I noticed an error in `README.rst`:

https://github.com/openwisp/openwisp-controller/commit/87c42ce4ccf39838475c2a70b4248314d84ad84f#diff-88b99bb28683bd5b7e3a204826ead112L73

`python setup.py develop` should be still run when using `pipenv`, using `pipenv run python setup.py develop`. What do you think?

Reply:
> @nemesisdesign I think I noticed an error in `README.rst`:
> 
> [87c42ce#diff-88b99bb28683bd5b7e3a204826ead112L73](https://github.com/openwisp/openwisp-controller/commit/87c42ce4ccf39838475c2a70b4248314d84ad84f#diff-88b99bb28683bd5b7e3a204826ead112L73)
> 
> `python setup.py develop` should be still run when using `pipenv`, using `pipenv run python setup.py develop`. What do you think?

@okraits I was following your tips, but installed version of `openwisp_users` by `pipenv run python setup.py develop` is  `0.1.10 ` which is older than version installed via github (it is `0.1.11a0`) and I still see the same error

Reply:
it's django-x509 which is causing this issue in this case